### PR TITLE
Tasks1.md - Todoist

### PR DESCRIPTION
**Pull request by aaronjae22**
This article describes Todoist options to export users data.

### CODE REVIEW

File: articles/tasks1.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+There are some third-party open source solutions that allow user information to be exported in JSON and CVS formats, but eventually, the information in these formats does not allow import compatibility in other similar services.
+
+## Todoist to Things
+
+Some services, such as Things, allow the user to migrate their information from Todoist using the Todoist public API (https://developer.todoist.com/sync/v9/#get-all-projects), but it can result in a complicated and tedious process for the user because queries must be performed to obtain the information requested.
+
+## Todoist to CSV or JSON
+This (third-party open source)[https://darekkay.com/todoist-export/] allows the user to backup Todoist data as CSV or JSON.
+
+* Once you select the format in which you want your data to be exported, you need to authorize and sign in with your Todoist account in order for the tool to begin the backup process.
```
Comment (lisad) [inline articles/tasks1.md:21]: Does this require sharing a password with the tool?  If it does we should warn about that

**Review by lisad — APPROVED** [review]

File: articles/tasks1.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: Transfering and exporting Todoist tasks
+datatype: Tasks
+sources: Todoist
+destination: ["Google Sheet," "CSV", "JSON", "Things"]
```
[the remark this answers is not in the document]
Comment (aaronjae22) [reply articles/tasks1.md:5]: That will definitely be better for sure. I'm on it

File: articles/tasks1.md (hunk 1 above)
Comment (aaronjae22) [reply articles/tasks1.md:21]: Yes, it does. I will write something to warn about it.

> Export for Todoist will be able to receive the following permissions:
> 
> View your user info
> Read your app data, including projects, tasks, notes, labels, and filters

**Comment by lisad** [comment]
As long as it's not getting passwords, I don't think it's actually worth a
warning.  I just wasn't sure where the user was signing in - do they sign
in on the todoist site, which is fine, or sign into their todoist account
via the tool, which merits a warning.

**Comment by aaronjae22** [comment]
It seems you sign in into the todoist account via the tool
image

**Pull request merged**